factor/cohort/settings.php

```diff
@@ -38,10 +38,11 @@
     new lang_string('settings:weight', 'tool_mfa'),
     new lang_string('settings:weight_help', 'tool_mfa'), 100, PARAM_INT));
 
-$cohorts = cohort_get_all_cohorts();
+global $DB;
```

Comment:
the global call isnt required here, as this file is direct included from a context where it should already be declared. This is only required inside of methods or other scope changes from the global scope.

Reply:
https://github.com/catalyst/moodle-tool_mfa/pull/452/commits/764f1e4e7c5ccc10dae1832f5620858003cf3ad2